mtest/mscore/palette/tst_palette.cpp

```diff
@@ -68,6 +68,7 @@ void TestPaletteModel::testDuplicateItemNames()
                   if (name.key().endsWith(" repeat sign") // repeat barlines in "Barlines" and "Repeats & Jumps" palette
                       || name.key() == "Open" // articulations in "Articulations" and channel switch text in "Text" palette
                       || name.key() == "Line" // bracket type in "Brackets" and line type in "Lines" palette
+                      //|| name.key() == "Caesura" // 2 different Caesuras in the "Breaths & Pauses" palette (so far Master palette only)
```

Comment:
needed only if and when that symbol gets added to the basic or advanced workspace